go.mod

```diff
@@ -89,7 +89,7 @@ require (
 	k8s.io/cli-runtime v0.23.1
 	k8s.io/client-go v0.23.1
 	k8s.io/klog/v2 v2.40.1
-	k8s.io/kubernetes v1.22.6
+	k8s.io/kubernetes v1.22.7
```

Comment:
@castelblanque, unfortunately, since we have a `replace` section, bumping it up does not effectively upgrade this dep. We have to manually change it in this `replace` section so that the change takes effect.
Usually, what I do is commit those changes and push them to this dependabot's PR.

Reply:
You are right, thanks for sharing.